doc/classes/Color.xml

```diff
@@ -348,7 +348,7 @@
 			Brown color.
 		</constant>
 		<constant name="burlywood" value="Color( 0.87, 0.72, 0.53, 1 )">
-			Burly wood color.
+			Burlywood color.
```

Comment:
Should be "Burly wood"

Reply:
Favouring Burlywood here since it's the real word.

Reply:
I'll go with Burly wood for consistency.